style.css

```diff
@@ -1,413 +1,894 @@
-@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,600;1,100;1,200;1,300&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@200;300;400;500;600;700;800;900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
 
 body {
-  font-family: "Poppins", sans-serif;
+  background-color: yellow;
+  padding: 0;
+  margin: 0;
+}
+
+.hamburger {
+  background: #3c3a39;
+  height: 8%;
+  padding: 20px 10px 5px 10px;
 }
 
-.header {
+.head {
   display: flex;
-  align-items: center;
   justify-content: space-between;
-  padding: 18px 24px;
-  position: relative;
+  align-items: center;
+  height: 100%;
+  padding: 5px 12px;
 }
 
-.header > .header-logo {
-  text-decoration: none;
-  font-weight: 700;
+.name-me {
+  font-family: "roboto", sans-serif;
+  font-style: normal;
+  font-weight: bold;
   font-size: 18px;
-  line-height: 20px;
-  text-align: center;
-  color: #6070ff;
-  padding: 14px 0;
+  color: white;
+  margin-left: 53px;
 }
 
-.header-menu-btn > * + * {
-  margin-top: 3px;
+section {
+  background: #1c1a19;
+  color: whitesmoke;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
```

Comment:
[OPTIONAL] The padding value that you used in the `section` selector is affecting all the sections of your page. Kindly remove the padding from the `section` selector and define a class name for each section with its padding.

<img width="285" alt="image" src="https://user-images.githubusercontent.com/77190203/155325276-80f1af00-d5e4-408c-bffa-c2c7c6a4040e.png">